src/Bird.ts

```diff
@@ -37,17 +39,17 @@ export class Bird extends NPC {
     private state = BirdState.WAITING_LEFT;
     private jumpTimer = 0;
 
-    public constructor(scene: GameScene, x: number, y: number) {
-        super(scene, x, y, 28, 24);
-        this.minAltitude = y;
+    public constructor(scene: GameScene, position: Point) {
+        super(scene, position, new Size(28, 24));
+        this.minAltitude = position.y;
         this.conversation = new Conversation(conversation, this);
 
         this.doubleJumpEmitter = this.scene.particles.createEmitter({
-            position: {x: this.x, y: this.y},
-            velocity: () => ({ x: rnd(-1, 1) * 90, y: rnd(-1, 0) * 100 }),
+            position: this.position,
```

Comment:
Müsste hier gecloned werden, ansonsten folgen die Partikel späteren Änderungen an this.position